contracts/cosmwasm-vm/cw-mock-dapp/src/contract.rs

```diff
@@ -1,11 +1,11 @@
 use std::str::from_utf8;
 
-use cw_common::xcall_types::network_address::NetworkAddress;
+use cw_xcall_lib::{network_address::NetworkAddress, xcall_msg::ExecuteMsg};
 
 use super::*;
 
 // version info for migration info
-const CONTRACT_NAME: &str = "crates.io:cw-mock-dapp";
+const CONTRACT_NAME: &str = "crates.io:cw-mock-dapp-multi";
 const CONTRACT_VERSION: &str = env!("CARGO_PKG_VERSION");
```

Comment:
I think this should still be "crates.io:cw-mock-dapp" if we are keeping both mock-dapp and mock-dapp-multi projects.